kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/lang/types/GenFnType.java

```diff
@@ -40,18 +41,15 @@ public GenFnType(List<Type> argsGen, Type returnGen) {
 
     @Override
     public boolean isInstanceOf(Object o) {
-        if (o instanceof FEELFunction) {
-            FEELFunction oFn = (FEELFunction) o;
-            List<List<Param>> signatures = oFn.getParameters().stream().filter(signature -> signature.size() == argsGen.size()).collect(Collectors.toList());
-            for (List<Param> signature : signatures) {
-                if (signature.size() == argsGen.size() && IntStream.range(0, argsGen.size()).allMatch(i -> argsGen.get(i).conformsTo(signature.get(i).type))) {
-                    return true;
-                }
+        if (o instanceof FEELFunction oFn) {
+            List<List<Param>> parameters = oFn.getParameters();
+            if(parameters.isEmpty()){
```

Comment:
```suggestion
            if (parameters.isEmpty()) {
```
Please use the same formatting style of the code base